src/main/java/org/opensearch/security/OpenSearchSecurityPlugin.java

```diff
@@ -2109,8 +2155,12 @@ public Collection<SystemIndexDescriptor> getSystemIndexDescriptors(Settings sett
             ConfigConstants.SECURITY_CONFIG_INDEX_NAME,
             ConfigConstants.OPENDISTRO_SECURITY_DEFAULT_CONFIG_INDEX
         );
-        final SystemIndexDescriptor systemIndexDescriptor = new SystemIndexDescriptor(indexPattern, "Security index");
-        return Collections.singletonList(systemIndexDescriptor);
+        final SystemIndexDescriptor securityIndexDescriptor = new SystemIndexDescriptor(indexPattern, "Security index");
+        final SystemIndexDescriptor resourceSharingIndexDescriptor = new SystemIndexDescriptor(
+            ConfigConstants.OPENSEARCH_RESOURCE_SHARING_INDEX,
```

Comment:
Nit: Do we want to add a TODO to support custom resource sharing names?

Reply:
Not right now no. Maybe in the next iteration if there is a request for it.